src/Blocks/BlockSponge.h

```diff
@@ -113,7 +113,7 @@ class cBlockSpongeHandler :
 				if (seed.m_Depth > 0)
 				{
 					// If this block was water, and we haven't yet gone too far away,
-					// add itś neighbors to the queue to check.
```

Comment:
Czech?

Reply:
No, we don't have `ś` in the language. Only `š`.